clusterloader2/pkg/measurement/util/pods.go

```diff
@@ -23,6 +23,8 @@ import (
 	"k8s.io/apimachinery/pkg/util/sets"
 )
 
+//go:generate stringer -type=status
```

Comment:
How does it work, does it require some tooling to regenerate?

Reply:
https://pkg.go.dev/golang.org/x/tools/cmd/stringer 
This can be regenerated via:
`go generate` or `stringer -type=status` in utils directory when you have stringer tool installed.

Reply:
I’m afraid that lack of boilerplate will block this approach. Each file in the repo needs the header with some legal stuff. can we add it here?